syserol/figures/figure3.py

```diff
@@ -1,15 +1,29 @@
 """
 This creates Figure 3.
 """
-
+import numpy as np
 from .common import subplotLabel, getSetup
+from syserol.impute import evaluate_missing
 
 
 def makeFigure():
-    """Get a list of the axis objects and create a figure"""
+    """ Evaluate Handling of Missing Values in Decomposition """
     # Get list of axis objects
-    ax, f = getSetup((10, 10), (1, 1))
+    ax, f = getSetup((10, 10), (2, 1))
+    
+    Averages, Sums = evaluate_missing()
+    
+    x = np.arange(1,10)
+    ax[0].plot(x, Averages)
+    ax[0].set_ylabel("Difference", fontsize = 12)
```

Comment:
We should just plot the sum of squared difference.

Reply:
Let me know if this is what you were looking for (I think just squaring the final sums would be incorrect?)